docs/src/man/comparisons.md

```diff
@@ -0,0 +1,27 @@
+The table compares the main functions of DataFrames with R dplyr and Stata
+| DataFrames       | dplyr | Stata|
+|:------------|:------------|:------------|
+|`combine(df, :x => mean)`  | `summarize(mean(x))`    | `collapse (mean) x `|
+|`transform(df, :x => mean)`   | `mutate(mean(x))`    | `egen x_mean = mean(x)`|
+|`select(df, :x, :y)`   | `select(df, x, y)`  | `keep x y` |
+|`sort(df, :x)`   | `arrange(df, x)`    | `sort x`|
+|`filter(:x => x -> x >= 1, df)`   | `filter(df, x >= 1)`  | `keep if x >= 1` |
+
+Here are more complicated examples for `combine`/`transform`/`select`:
+| DataFrames       | dplyr | Stata|
+|:------------|:------------|:------------|
+|`combine(df, :x => mean => :x_mean)`   | `summarize(df, x_mean = mean(x))`    | `collapse (mean) x_mean = x`|
+|`combine(df, :x => x -> maximum(x) - minimum(x))`   | `summarize(df, max(x) - min(x))`    | |
```

Comment:
you want to say that this is not achievable in Stata at all - if this is the case it is strange (but I do not know Stata)

Reply:
also - out of couriosity - what would be the equivalent of:
```
combine(:x => x -> (extremum=["max", "min"], value=[maximum(x), minimum(x)]), df)
```
in dplyr and Stata?

Reply:
In Stata, `combine` only accepts a specific set of functions. To  do more complicated things, you would have to do a combination of `transform!`and `filter!`:
```
egen out = cov(x, y)
keep if _n == 1
```

Reply:
added